tests/benchmarking.py

```diff
@@ -25,6 +25,7 @@ def benchmark_model(id, path, io, n_inputs, n_outputs, n_runs=1):
         elif result == 0:
             print("[Warning] Measured time is 0. Perhaps trying to measure a very small time."+
                   "There is a known issue with time library in Windows.")
+            model_times.append(result)
```

Comment:
I wasn't storing result as its 0 here. Any reason behind?

Reply:
you divide by the length a few lines later, that threw a DivByZero exception